### PR TITLE
Make it compatible with RN 0.60.0

### PR DESCRIPTION
**Pull request by RaeesBhatti**

### CODE REVIEW

**Comment by zxcpoiu** [comment]
@raeesbhatti @rnunley 

Hi, does this a breaking change? I mean, after this merged, can user still use this plugin in 0.59 and below?

**Comment by raeesbhatti** [comment]
@zxcpoiu Actually I'm not sure about that, it would be great if you could test that and let us know.

File: RNVoipPushNotification.podspec [hunk 1]
```diff
@@ -14,5 +14,5 @@ Pod::Spec.new do |s|
   s.requires_arc        = true
   s.platform            = :ios, "8.0"
   s.source_files        = "ios/RNVoipPushNotification/*.{h,m}"
-  s.dependency 'React/Core'
+  s.dependency 'React-Core'
```
Comment (zxcpoiu) [inline RNVoipPushNotification.podspec]: Seems like it's a breaking change, could you test if works for you if you change it

from:
`s.dependency 'React-Core'`
to
`s.dependency 'React'`
?

Comment (ThomasLarge) [reply RNVoipPushNotification.podspec]: I can confirm it works with both but it looks like other packages us `s.dependency 'React'` as I think it's backwards compatible

**Pull request merged**